test/dlc_tests/sanity/test_canary_integration.py

```diff
@@ -54,3 +57,21 @@ def _run_canary_pull_test(region, image_type):
         LOGGER.info(f"Canary image {image} is available")
         # Do not remove the pulled images as it may interfere with the functioning of the other
         # tests that need the image to be present on the build machine.
+
+
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.skipif(
+    not is_deep_canary_context(), reason="This test only needs to run in deep-canary context"
+)
+@pytest.mark.deep_canary
```

Comment:
Can you provide a description on this test? Is this the only test you'll be marking as deep canary for now?

Reply:
How about:
```
@pytest.mark.deep_canary(
    "Reason: This test acts as a basic smoke check that deep-canary tests work in a region"
)
```
?

Yes, this is the only test that I'm marking as deep canary for now.